### ISSUE TITLE
Persistent error syncing drafts

### ISSUE THREAD

**Issue by loleg** [djmaze/tiddlywiki-docker#2]
I haven't seen this in the stock Node.js app, so assume it must be something in the filesystem permissions or so. I regularly get these warnings popping up over my wiki:

```
syncer-server - 16:48:20 10 3 2015
Sync error while processing 'Draft of 'MyTiddler: Error: ENOENT, unlink '/var/lib/tiddlywiki/mywiki/tiddlers/Draft of 'MyTiddler'.tid'
```

**Comment by loleg** [comment]
There has been a [partial patch](https://github.com/Jermolene/TiddlyWiki5/commit/550ad4a1a914bbd68aa6cee2be11da0f55d903c2) to address this issue, but @Jermolene is asking for help diagnosing the root cause. It is affecting other Node.js installations so is not specific to this Docker image. I've installed it on my instance using the [Dockerfile in my branch](https://github.com/loleg/tiddlywiki-docker/tree/run-master) to pull from master instead of npm.

**Comment by djmaze** [comment]
Thanks, I am having the same problem from time to time, when connection is lost. I suppose there will soon be a release containing at least this partial fix, so I'd like to stay on released versions for now.

**Comment by 0atman** [comment]
Has this been fixed?

**Comment by djmaze** [comment]
The Docker image hasn't been built for about a year. I just triggered a new build so the new image should contain the latest Tiddlywiki version. I am not sure if they finally fixed the error correctly upstream.

**Comment by 0atman** [comment]
I rebuilt it for my fork, and it works still, so no fear about that! Are you still using TW?

**Comment by djmaze** [comment]
@0atman Yes, just didn't update it for a long time. Am quite happy with it. Best app for taking personal notes. Although I am quite curious what has changed since back then!

**Comment by 0atman** [comment]
(I hope you don't mind conversing off-topic)
I am totally new to TW, hence wanting to set up a server for myself. I've spotted the codepen plugin, which is great so far!
